packages/ui-extensions/src/surfaces/checkout/api/standard/standard.ts

```diff
@@ -350,79 +349,6 @@ export interface AppMetafieldEntry {
 
 export type Version = 'unstable';
 
-/**
- * This defines the i18n.translate() signature.
- */
-export interface I18nTranslate {
```

Comment:
@kumar303 @marvinhagemeister do you know if any of the docs extraction stuff is depending on the exact file structure of our packages? This is on the shared UI extension package, so it won't impact checkout yet, but we will need to make sure it works as part of versioning.